tests/integration/README.md

```diff
@@ -1,25 +1,18 @@
 ## Integration Testing
+Integration test verifies the functionality of EBS CSI driver as a standalone server outside kuberentes. It exercises the lifecycle of create, attach, stage, mount volume and the reverse operations. And it verifies data can be written onto an EBS volume without any issue.
 
-### Requirements 
+## Run Integration Tests Locally
+Integration test is executed using [aws-k8s-tester](https://github.com/aws/aws-k8s-tester) which is CLI tool for k8s testing on AWS. With aws-k8s-tester, it automates the process of provisioning EC2 instance, pull down and build EBS CSI driver, running the defined integration test and sending test result back. See aws-k8s-tester for more details about how to use it.
```

Comment:
`The integration test`
`provisioning an EC2 instance, pulling down and building`